std/conv.d

```diff
@@ -5692,7 +5692,7 @@ auto toChars(ubyte radix = 10, Char = char, LetterCase letterCase = LetterCase.l
             enum SHIFT = 4;
         else
             static assert(0);
-        struct Result
+        static struct Result
```

Comment:
Without "static" the struct gets an unneeded context pointer, because it's declared in a function. opSlice does void initialization and doesn't copy the context pointer over, so comparisons would fail.

Since the context pointer isn't needed, getting rid of it seems to be obvious solution here. The base 10 Result above is also static.